strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/jobs/AbstractCronJobsRegistry.java

```diff
@@ -0,0 +1,37 @@
+package org.carlspring.strongbox.cron.jobs;
+
+import java.lang.reflect.Modifier;
+import java.util.Set;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.ImmutableSet;
+import org.reflections.Reflections;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Component
+class AbstractCronJobsRegistry
```

Comment:
naming seems confusing because we already have the `CronJobsDefinitionsRegistry`, also `Abstract` commonly means abstract class.

Reply:
> also Abstract commonly means abstract class.

yeah, but it's a registry of `AbstractCronJob` cron jobs :)

Reply:
I've renamed it to `CronJobsRegistry`

Reply:
> naming seems confusing because we already have the CronJobsDefinitionsRegistry

`CronJobsDefinitionsRegistry` is a registry of `CronJobDefinition`. `CronJobsDefinitionsRegistry` uses `CronJobsRegistry` in construction